src/app/shared/file.service.spec.ts

```diff
@@ -29,14 +27,14 @@ describe('FileService', () => {
     });
   });
 
-  it('should escape correctly', inject([FileService], (fileService: FileService) => {
-    expect(fileService.escapeEntities('python <<CODE')).toEqual('python &lt;&lt;CODE');
-    expect(fileService.escapeEntities('hello')).toEqual('hello');
-    expect(fileService.escapeEntities('<danger>Thing\'s & "things"</danger>'))
-      .toEqual('&lt;danger&gt;Thing&#039;s &amp; &quot;things&quot;&lt;/danger&gt;');
-    expect(fileService.escapeEntities('')).toEqual('');
-    expect(fileService.escapeEntities(null)).toEqual(null);
-  }));
+  // it('should escape correctly', inject([FileService], (fileService: FileService) => {
```

Comment:
Comment should indicate under what circumstances this would be uncommented, if applicable or deleted otherwise.